core/src/main/java/org/apache/carbondata/scan/result/vector/CarbonColumnVector.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.carbondata.scan.result.vector;
+
+import org.apache.spark.sql.types.Decimal;
```

Comment:
The interface would be used in Carbon layer to set the batch data and implementation class `ColumnarVectorWrapper` is in spark layer to avaoid spark dependency. So there would not be any spark dependency in carbon layer.